formbricks/formbricks
# feature/add consent question

## What does this PR do?



Adds the consent question type that can be used for terms&conditions etc.


![image](https://github.com/formbricks/formbricks/assets/42251569/9f11fcb3-26b9-4b3b-a77d-172812ce9918)
![image](https://github.com/formbricks/formbricks/assets/42251569/e97646c9-950d-4de3-88e6-776ce00792a9)
![image](https://github.com/formbricks/formbricks/assets/42251569/a19432e1-ed35-47f0-82dc-98b86e4f3a13)


## Type of change




## How should this be tested?



- Add consent question type to link survey and in-product survey and check UI
- Submit both checked and skipped and check summary

## Checklist
